doc/user/content/sql/create-sink/kafka.md

```diff
@@ -192,13 +414,38 @@ CREATE CONNECTION csr_basic_http
 
 ### Creating a sink
 
-When sinking into Kafka, Materialize will write all the changes from the
-specified source, table, or materialized view into the sink topic. If the topic
-does not exist, Materialize will attempt to create it.
+#### Upsert envelope
 
-{{< note >}}
-{{% kafka-sink-drop  %}}
-{{</ note >}}
+{{< tabs tabID="1" >}}
+{{< tab "Avro">}}
+
+```sql
+CREATE SINK avro_sink
+  FROM <source, table or mview>
+  INTO KAFKA CONNECTION kafka_connection (TOPIC 'test_avro_topic')
+  KEY (key_col)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection
+  ENVELOPE DEBEZIUM
+  WITH (SIZE = '3xsmall');
+```
+
+{{< /tab >}}
+{{< tab "JSON">}}
+
+```sql
+CREATE SINK json_sink
+  FROM <source, table or mview>
+  INTO KAFKA CONNECTION kafka_connection (TOPIC 'test_json_topic')
+  KEY (key_col)
+  FORMAT JSON
+  ENVELOPE DEBEZIUM
+  WITH (SIZE = '3xsmall');
```

Comment:
Same comment as before. If we want to start discouraging linked clusters, we might want to use `IN CLUSTER` here.

Reply:
Done.